pkg/builtin/manifest.yml

```
@@ -1584,3 +1584,75 @@ extensions:
             - id: layerCamera
               title: Camera position
               type: camera
+  - id: cluster
+    name: Cluster
+    type: cluster
+    description: defines how layers are clustered together and displayed on earth.
+    schema:
+      groups:
+        - id: default
+          title: Cluster
+          fields:
+            - id: cluster_name
+              type: string
+              title: Name
+              description: Sets the name of the cluster.
+            - id: cluster_pixelRange
+              type: number
+              description: Sets the minimum range between layers to get clustered together.
+              title: Pixel range
+              defaultValue: 15
+              min: 1
+              max: 200
+            - id: cluster_minSize
+              type: number
+              title: Minimum cluster size
+              description: the minimum number of layers that can be clustered.
+              defaultValue: 3
+              min: 2
+              max: 20
+            - id: cluster_sizeType
+              title: Size type
```

```suggestion
            - id: cluster_maxSize
              title: Max cluster size
```

I think size type is hard to understand the meaning and a bit confusing for me.